src/components/Card.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { SyntheticEvent, useRef, useState } from "react";
+import styles from "./Card.module.css";
+import { TextInput } from "./TextInput";
+import { SmallIconButton } from "./SmallIconButton";
+import { TextOutput } from "./TextOutput";
+import { BigButton } from "./BigButton";
+
+interface CardProps {
+  /**text on side A */
+  faceValue?: string;
```

Comment:
**nitpick:** Why these props are optional? Btw, they should have more descriptive names, like `frontValue`, `backValue`.